### DISCUSSION TITLE
Secret vs Config Context?

### DISCUSSION THREAD

**Discussion by Joshi425** [DanSheps/netbox-secretstore#27] [General]
Hello there,

why are secrets different then Config Context? Why are they limited to one Device? 
If I have 5 webservers all 5 need the private key to my certificate. Am I supposed to create the same secret 5 times to assign it to all my servers?

IMO Secrets should be like Config Contexts but encrypted.

Have a nice day,

Joshi

**Comment by jameskirsop** [comment]
@Joshi425 would suggest you 👍 this issue: https://github.com/DanSheps/netbox-secretstore/issues/31 which seems to be requesting what you're wanting to solve here.
